cli-tools/src/integration/java/com/radixdlt/integration/targeted/capabilities/NodeCapabilityTests.java

```diff
@@ -187,7 +187,7 @@ public void ensure_messages_for_an_enabled_capability_are_processed() {
           m -> messagesReceived.add("node1-" + m.message().getClass().getSimpleName()));
 
       // Have the expected messages been received?
-      var result = waitForMessagesReceived(expectedResultMap, messagesReceived, 3);
+      var result = waitForMessagesReceived(expectedResultMap, messagesReceived, 10);
```

Comment:
This test flaked out on me, probably because genesis is a little slower, so I increased the timeout